src/genericSchemeRegistry/schemes/BalancerPoolManager.json

```diff
@@ -0,0 +1,341 @@
+{
+  "name": "BalancerPoolManager",
+  "addresses": {
+    "main": [
+      ""
+    ],
+    "kovan": [
+      "0x5086d91858a5aac4a770212720b297f38e0dd403"
+    ],
+    "private": [
+      ""
+    ]
+  },
+  "actions": [
+    {
+      "id": "setPublicSwap",
+      "label": "Set PublicSwap parameter to true/false",
+      "description": "Pause/unpauses the pool",
+      "notes": "https://github.com/PrimeDAO/monorepo/blob/master/packages/contracts/contracts/schemes/BalancerProxy.sol",
+      "fields": [
+        {
+          "label": "Set public swap to",
+          "name": "public swap",
+          "placeholder": "Boolean (true)"
+        }
+      ],
+      "abi": {
+        "constant": false,
+        "inputs": [
+          {
+            "internalType": "bool",
+            "name": "publicSwap",
+            "type": "bool"
+          }
+        ],
+        "name": "setPublicSwap",
+        "outputs": [],
+        "payable": false,
+        "stateMutability": "nonpayable",
+        "type": "function"
+    }
+    },
+    {
+      "id": "setSwapFee",
+      "label": "Set Swap Fee",
+      "description": "Update the Swap Fee",
+      "notes": "https://github.com/PrimeDAO/monorepo/blob/master/packages/contracts/contracts/schemes/BalancerProxy.sol",
+      "fields": [
+        {
+          "decimals": 18,
```

Comment:
Is decimals what causes the amount to be converted to wei?

Reply:
I think so, yeah